packages/augur-core/source/libraries/ContractDeployer.ts

```diff
@@ -245,7 +245,7 @@ Deploying to: ${networkConfiguration.networkName}
         return contract.address;
     }
 
-    private async uploadAllContracts(serial=false): Promise<void> {
+    private async uploadAllContracts(serial=true): Promise<void> {
```

Comment:
@nuevoalex how do you feel about defaulting to serial uploads? I haven't been able to deploy to ganache from a separate process (like flash run deploy) when this is run in parallel because of nonce errors. I added the `serial` parameter and code so I could easily tweak it for that kind of test but I haven't worked out how to actually pass in the right argument here.

I assume running in parallel is fantastic with geth and such because that fits many txs into one block.

Reply:
as long as its not extremely painfully slow thats fine. The purpose of the Fast Wallet is supposed to be to allow non serial uploads but I've also seen the occasional nonce error so probably better to just suck it up. Not doing too much iteration with the integration tests anyway.